lib/components/user/account-page.tsx

```diff
@@ -34,14 +49,22 @@ class AccountPage extends Component {
     }
   }
 
-  componentDidMount () {
+  _hidePopup = () => {
+    const { setPopupContent } = this?.props
+    if (setPopupContent) setPopupContent(null)
+  }
+
+  componentDidMount() {
     this._checkAccountCreated()
   }
 
-  render () {
-    const {children, subnav = true} = this.props
+  render() {
+    const { children, popupContent, subnav = true } = this.props
     return (
-      <AppFrame SubNav={subnav && SubNav}>
+      // @ts-expect-error TODO: add typing for SubNav
+      <AppFrame SubNav={SubNav}>
```

Comment:
The `subNav` boolean prop is still needed. It is used to hide the "My account" header and the "Trips/Settings" tabs when saving a trip.